server/restivus-swagger.js

```diff
@@ -46,19 +49,9 @@ Restivus.prototype.addSwagger = function(path) {
             _.each(endpoints, function(endpoint) {
```

Comment:
Try using `_.map()` here to generate a new array, as opposed to modifying the values of the original array.